docs/pages/database-access/auto-user-provisioning/mysql.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: MySQL Automatic User Provisioning 
+description: Configure automatic user provisioning for MySQL.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="14.1"
+>
+  Automatic user provisioning for MySQL is available starting from Teleport
+  14.1.
+</Details>
+
+## Prerequisites
+
+- Teleport cluster with a configured [self-hosted MySQL](../guides/mysql-self-hosted.mdx)
+  or [RDS MySQL](../guides/rds.mdx) database.
+- Ability to connect to and create user accounts in the target database.
+
+<Admonition type="note" title="Supported versions">
+Automatic user provisioning are not compatible with MySQL versions lower than
+8.0.
+</Admonition>
+
+## Step 1/3. Configure database admin
+
+(!docs/pages/includes/database-access/auto-user-provisioning-configure-admin.mdx!)
+
+Teleport use the same authentication mechanism when connecting as an admin user
+as for regular user connections: X.509 for self-hosted databases and AWS IAM
+for RDS.
+
+The admin user must have privileges within the database to create users and
+grant them privileges. The admin user must also have privileges to monitor user
+processes and role assignments.
+
+In addition, a database/schema is required for the admin user to logs into by
```

Comment:
What does "database/schema" mean here?

Reply:
Ya. It's a database inside the MySQL server, like the <db-name> in `CREATE DATABASE <db-name>` or `tsh db connect --db-name <db-name>`.  Schema and database are synonym for MySQL. 

I thought it would be unclear to say a `database is required xxx`. Maybe I should just leave it as `schema`?

Reply:
I think `a schema is required` works!